### ISSUE TITLE
Upload Driver accept method not working

### ISSUE THREAD

**Issue by ralliart2004** [MasoniteFramework/core#696]
I have tried to allow a range of file types in the upload driver with the accept method, however, it wont let me upload anything except the defaults. I end up having to comment out the validate_extension check in the UploadDiskDriver class and UploadS3Driver class to be able to upload anything outside of the defaults. 

There are many file types and making a list of all is insane. I would recommend in this fix to also add a wildcard option to the accept method and let the composer of the code deal with the security concern if they opt-in.

**Comment by josephmancuso** [comment]
do you still have the code snippet to how you were using this method? I did some testing myself recently and everything looks fine on my end.

**Comment by ralliart2004** [comment]
I have moved passed this a while back. I could test this out on a new project I am about to start in Masonite. I will let you know if this problem still persists in a newer version.

**Comment by josephmancuso** [comment]
```python
        upload.driver('disk').accept("mp4").store(
            request.input('file'),
            location=temp_dir,
            filename=chunk_filename
        )
```

![image (1)](https://user-images.githubusercontent.com/20172538/58831176-16205480-861a-11e9-8285-e694f429834b.png)

**Comment by josephmancuso** [comment]
should be fixed in 2.1.30
